app/models/onboarding_form_service.rb

```diff
@@ -113,6 +113,9 @@ def generate_ticket_body(onboarding_form)
                 Department:
                 #{ value_or_default(onboarding_form.contact_details_department) }
 
+                Level of assurance:
```

Comment:
We should move this up to the same place as we display it in the form too, so it's easier to know where to look for it.